api/Port.go

```diff
@@ -276,3 +285,118 @@ func (vrsConnection VRSConnection) processUpdates(updates *libovsdb.TableUpdates
 	}
 	return nil
 }
+
+// AddPortToAlubr0 adds Nuage port to alubr0 bridge
+func AddPortToAlubr0(ovs *libovsdb.OvsdbClient, intfName string, entityName string, entityUUID string) error {
```

Comment:
Indeed. I was working on making the same change while you added the comment. Thanks for pointing this out.